woodwork/deserializers/parquet_deserializer.py

```diff
@@ -1,11 +1,79 @@
-from woodwork.deserializers.deserializer_base import Deserializer
+import json
+import os
+import tarfile
+import tempfile
+from pathlib import Path
+
+import pyarrow as pa
+
+from woodwork.deserializers.deserializer_base import Deserializer, _check_schema_version
+from woodwork.s3_utils import get_transport_params, use_smartopen
+from woodwork.utils import _is_s3, _is_url
 
 
 class ParquetDeserializer(Deserializer):
     """Deserialize Woodwork table from serialized data in parquet format."""
 
     format = "parquet"
 
+    def deserialize(self, profile_name, validate):
```

Comment:
Nit: Can we add docstrings for these methods?

Reply:
I had originally left docstrings off of all these methods in the serializer/deserializer classes since they are defined on the `Serializer` and `Deserializer` base classes and thought it might be redundant to include them on the subclasses (and something else that would need to be maintained if the base docstrings changed).

I'm not opposed to adding them here though - especially for methods that override the base class method and might behave slightly differently.

Reply:
Great point, I missed that! It could just be nice to add them when there are additional args or different behavior to clarify how it differs from the base class.